storybookjs/storybook
# Fix react-native-vanilla to actually run storybook and document how to run it

Issue:

react-native-vanilla example app was never configured to run Storybook RN

## What I did
reconfigured it to run Storybook RN and added documentation.

## How to test

Is this testable with jest or storyshots?

No. This breaks jest tests https://github.com/storybooks/storybook/issues/2273 because we have been running tests for the original example provided by react-native, not our Storybook code. We need to think about how to run react-native tests because it doesn't seem too obvious to me.

Does this need a new example in the kitchen sink apps?

perhaps?

Does this need an update to the documentation?

this includes it

If your answer is yes to any of these, please make sure to include it in your PR.
